shakenfist/net.py

```diff
@@ -99,7 +104,33 @@ def subst_dict(self):
     def persist_floating_gateway(self):
         db.persist_floating_gateway(self.uuid, self.floating_gateway)
 
+    def is_okay(self):
+        """Check if network is created and running"""
+        # TODO(andy):This will be built upon with further code re-design
+
+        okay = self.is_created()
+
+        if is_network_node():
+            okay &= self.is_dnsmasq_running()
+
+        return okay
+
+    def is_created(self):
+        """Attempt to ensure network has been created successfully"""
+
+        subst = self.subst_dict()
+        out, _ = processutils.execute('ip link show %(vx_bridge)s' % subst,
```

Comment:
Why not use util.check_for_interface here?

Reply:
I want to check that it exists and is up. Since the "up" is a separate step.

Reply:
When we are big boys we should add the up as an option on that util method. Not blocking you for now though.